joomla/joomla-cms
# New show_info option (hide/show) in com_contact and some minor changes

#### What this PR does?
1. Adds a new option to com_contact show_info. This option allows for the administratator to choose if they want to show the contact info. This is very useful, for instance, in cases when we only want to show a feedback form and not the contact information.
2. Normalize show_tags option across all com_contact xml files (the option was missing in some xml).
3. Change order of contact "show miscellaneous information" option in all xml files to appear before show articles option. This is done because show miscellaneous information it's a block of information (with tab, slider or heading, depending on the layout you choose) and not only a field like telephone, e-mail, etc. Also adds "Show " prefix to that option language variable.
#### How to test
1. Use a fresh Joomla install with sample items.
2. Create a contact with default options
3. Create a menu item to show the contact
4. Check the contact in the frontend. All should be fine
5. Play with options in Global Contact Component, local options in the contact you created and in the contact option in the menu item you created and check everything is fine.
6. Test the new show info option (by setting it to Hide) and check that the contact information doesn't appear in the frontend.
#### B/C

None that i think of.
#### More info

In future PR's i plan to normalize all com_contact XML files for better code readability, add showons, resolve some bugs and improve forms/tables accessibility.
